Lib/typing.py

```diff
@@ -3288,8 +3287,7 @@ def decorator(cls_or_fn):
     return decorator
 
 
-
-def override(method: F, /) -> F:
+def override[F: Callable[..., Any]](method: F, /) -> F:
```

Comment:
I just find this one significantly less readable, I'm afraid. The information density's too high for me. Don't want to block it if others disagree, though.

Reply:
At least the information is close by, instead of in a TypeVar definition hundreds of lines away.

Reply:
Maybe make a private type alias `type _Func = Callable[..., Any]` and use `def override[F: _Func]...`?

Reply:
> Maybe make a private type alias `type _Func = Callable[..., Any]` and use `def override[F: _Func]...`?

I'd like that much more, yeah

Reply:
Gives me a chance to use a `type` statement too :)

(We can't do that for the existing type alias `Text` because people might be doing `isinstance(something, Text)`, which won't work with the `type` statement.)